debian/jitsi-meet-prosody.postinst

```diff
@@ -4,6 +4,7 @@
 # see: dh_installdeb(1)
 
 set -e
+set -x
```

Comment:
This is temporary, right?

Reply:
Ups, I forgot to drop it. Yeah it is me testing the builds.